src/closures/constant_smagorinsky.jl

```diff
@@ -0,0 +1,89 @@
+Base.@kwdef struct ConstantSmagorinsky{T} <: ScalarDiffusivity{T}
+               C :: T = 0.23
+         prandtl :: T = 1.0
+    ν_background :: T = 1e-6
+    κ_background :: T = 1e-7
```

Comment:
Would `ν_molecular` and `κ_molecular` be better names or is it generally not true?

Reply:
I think we should prefer to agnostic about the meaning of these variables. I think a common usage would be `ν_background = 0`, so calling it 'molecular' is confusing. 

Probably it should be 0 by default too...

Reply:
That makes sense.